Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1935,6 +1935,8 @@ def expandMapping(self,seqList,mapping,index=None):
     def prepare_DQM(self, sequence = 'DQMOffline'):
         # this one needs replacement
 
+        # any 'DQM' job should use DQMStore in non-legacy mode (but not HARVESTING)
+        self.loadAndRemember("DQMServices/Core/DQMStoreNonLegacy_cff")
```

Comment:
@schneiml this essentially sets always ```DQMStore.enableMultiThread = True```, and it should not change behaviour for multi-threaded workflows in my understanding, and not prevent HARVESTING to run anyway, right?

Reply:
This is completely independent of HARVESTING, which should be configured somewhere else.

And yes, all serious workflows run with `enableMultiThread = True` anyways, so it makes sense to turn it on even on single-threaded workflows to get consistent behavior.